.github/workflows/runbook-freshness-check.yaml

```
@@ -0,0 +1,237 @@
+        required: false
+        type: string
+        default: ''
+
+jobs:
+  check-runbook-freshness:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Get full history for git log
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)